README.md

```diff
@@ -20,62 +20,88 @@ Minimal and straightforward when doing queries
 ```python
 from nmi_mysql import nmi_mysql
 ```
-- Initialization: Accepts two parameters, first being the config object and the second specifying if autoconnect to db is enabled. If set to false, call `con.connect()` 
+
+- Initialization: Accepts one parameter, which is the config object
 
 ```python
-try:
-    con = nmi_mysql.DB(conf, True)
-except Exception as err:
-    print(err)
+db = nmi_mysql.DB(conf)
+```
+
+- Connection
+
+```python
+db.connect()
 ```
+
 - Query execution: Accepts two parameters. The first is the query and the second is the list of parameters to be used. See example below
 
 ```python
-data = con.query(query, params)
+data = db.query(query, params)
 ```
+
 - Closing connection
 
 ```python
-con.close()
+db.close()
 ```
 
 **Sample config object**
+
 ```python
 conf = {
     'host': 'localhost',
     'user': 'root',
     'password':'',
     'db': 'mydb',
-    'port': 3306    
+    'port': 3306,
+    'max_pool_size': 20     # optional, default is 10
 }
 ```
 
 ##### SELECT operations
+
 ```python
 from nmi_mysql import nmi_mysql
 
-connection = nmi_mysql.DB(conf, True)
+db = nmi_mysql.DB(conf)
+db.connect()
 
-data1 = connection.query('SELECT * FROM mytable WHERE name = %s', ['ninz'])
-data2 = connection.query('SELECT * FROM mytable WHERE name IN (%s) AND age = %s', [['john', 'doe'], 10])
-
-connection.close()
+data1 = db.query('SELECT * FROM mytable WHERE name = %s', ['ninz'])
+data2 = db.query('SELECT * FROM mytable WHERE name IN (%s) AND age = %s', [['john', 'doe'], 10])
```

Comment:
Got it. I already updated the code.

It takes me time to fix support for multi-statement queries since sqlalchemy does not directly support this but pymysql does. See http://docs.sqlalchemy.org/en/latest/core/connections.html#multiple-result-sets.

Also, I'm having a problem on when to return affected rows since we already support multi-statement queries, I don't have a way to check each statement if it's a select or not. For now, I just return affected rows if there are no rows returned.

Here's the commit 4de0f558b573cf6480d5203190e565cbe7d72157.